tests/index.test.js

```diff
@@ -417,6 +419,50 @@ t.test('customService', t => {
     scope.done()
   })
 
+  t.test('directly call a service with custom header', async t => {
+    t.plan(3)
+    const otherServiceName = 'other-service'
+    const headers = {
+      [CLIENTTYPE_HEADER_KEY]: 'CMS',
+      [USERID_HEADER_KEY]: 'fjdsaklfaksldkksjkfllsdhjk',
+      [GROUPS_HEADER_KEY]: 'group-to-greet,group',
+      [BACKOFFICE_HEADER_KEY]: '',
+      header1: 'header1Value',
+    }
+    const scope = nock(`http://${otherServiceName}:3000`, {
+      reqheaders: headers,
+      badheaders: ['header2'],
```

Comment:
What is `'header2'` there?

Reply:
In the env variable ADDITIONAL_HEADER_TO_PROXY I add 2 header, header1 and header2. I rename two header in additionalHeader1 and additionalHeader2.
In this test I expect to pass only existent header

Reply:
OK

Reply:
ok